helm-charts/mend-renovate-ee/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+name: mend-renovate-enterprise-edition
+version: 6.5.0
+appVersion: 6.5.0
+description: Mend Renovate Enterprise Edition
+home: https://github.com/mend/renovate-ce-ee
+sources:
+  - https://github.com/renovatebot/renovate
+icon: https://app.renovatebot.com/images/logo.png
```

Comment:
@justo-mend do we want the renovate logo or the mend logo here?

Reply:
Mend logo would be better.

Reply:
This requires more investigation - to get the right perma-link for the right logo image. We will come back to this and update CE and EE charts together in a separate PR.